INSTALL.md

```diff
@@ -2,111 +2,93 @@
-* Install core + neural networks (PyTorch): `pip install "u8darts[torch]"`
-* Install core + AutoARIMA: `pip install "u8darts[pmdarima]"`
+#### LightGBMModel
+Install the `lightgbm` package (version 3.2.0 or more recent) using the [LightGBM install guide](https://lightgbm.readthedocs.io/en/latest/Installation-Guide.html)
 
-### Enabling Support for LightGBM
-
-To enable support for LightGBM in Darts, please follow the
-[installation instructions](https://lightgbm.readthedocs.io/en/latest/Installation-Guide.html) for your OS.
-
-#### MacOS Issues with LightGBM
-At the time of writing, there is an issue with ``libomp`` 12.0.1 that results in
-[segmentation fault on Mac OS Big Sur](https://github.com/microsoft/LightGBM/issues/4229).
-Here's the procedure to downgrade the ``libomp`` library (from the
-[original Github issue](https://github.com/microsoft/LightGBM/issues/4229#issue-867528353)):
-* [Install brew](https://brew.sh/) if you don't already have it.
-* Install `wget` if you don't already have it : `brew install wget`.
-* Run the commands below:
-```
-wget https://raw.githubusercontent.com/Homebrew/homebrew-core/fb8323f2b170bd4ae97e1bac9bf3e2983af3fdb0/Formula/libomp.rb
-brew unlink libomp
-brew install libomp.rb
-```
+### Enabling GPU support
+Darts relies on PyTorch for the neural network models.
+For GPU support, please follow the instructions to install CUDA in the [PyTorch installation guide](https://pytorch.org/get-started/locally/).
 
-#### Test environment Apple M1 processor
+### Using an emulated x64 environment on Apple Silicon.
+The installation of `darts` has been tested to work on Apple silicon (M1) (Python 3.10, OSX Ventura 13.2.1).
 
-We currently recommend to run Darts in an x_64 emulated environment on Mac computers with the Silicon M1 processor,
-instead of trying to install directly with native arm64 packages, many of the dependent packages still have compatibility 
-issues. The following is a proposed procedure, if you tested other procedures on similar hardware and they worked, 
-please let us know about them by opening an issue or by updating this file and opening a PR. 
+If you run into issues, you can always use rosetta to run in an (intel) emulated x64 environment:
 
 Before you start make sure that you have rosetta2 installed by running: 
-```
+
+```bash
 pgrep oahd
 ``` 
-If you see some process id you are ready to go, as internally rosetta is known as oah.
+
+If you see a process id you are ready to go, as internally rosetta is known as oah.
 
 If pgrep doesn't return any id then install rosetta2:
-```
+
+```bash
 softwareupdate --install-rosetta
 ```
 
 Below are the necessary instructions to create and configure the environment:
-- Start by installing conda (e.g., with miniforge : `brew install miniforge`).
+- Install conda if you haven't done so (e.g., with miniforge : `brew install miniforge`).
 - Create the x_64 environment : `CONDA_SUBDIR=osx-64 conda create -n env_name python=3.9 pip`
```

Comment:
update this to 3.10 to be coherent with the instruction above?

Reply:
👍